README.md

```diff
@@ -22,6 +22,9 @@ be logged on GitHub.
 
 ## Changelog
 
+* issue#:15 Fix webseer tab to not show items before schedule is created
+* issue#:15 Fix schedule delete invalid function
```

Comment:
There should not be two issues for the same number.  If the second is introduced by the first fix, it is merely a correct to the fix as long it is before a release.  After a release, a separate issue should raised to document the problem again.